purchase_stock_picking_return_invoicing/models/purchase_order.py

```diff
@@ -26,6 +26,11 @@ def _get_invoiced(self):
             'Product Unit of Measure'
         )
         for order in self.filtered(lambda x: x.state in ('purchase', 'done')):
+            # the field 'force_invoiced' is defined in the
+            # 'purchase_force_invoiced' module of OCA/purchase-workflow.
+            if hasattr(order, 'force_invoiced') and order.force_invoiced and \
```

Comment:
What about using `getattr` instead?
`if getattr(order, 'force_invoiced', None) and order.invoice_status == 'invoiced':`